source/reference/limitations.txt

```diff
@@ -34,9 +34,10 @@ General Limitations
   are properly configured.
 - Other clients must not write to the destination cluster while
   ``mongosync`` is running.
-- If write blocking is disabled, the client must :ref:`prevent writes
-  <c2c-api-start>` to the source cluster before starting the commit
-  process.
+- If you want to start the :ref:`commit <c2c-api-commit>`
+  process and you previously disabled :ref:`write blocking
+  <c2c-write-blocking>`, you must :ref:`prevent writes <c2c-api-start>`
+  to the source cluster before you start the commit process. 
```

Comment:
The "disabling" part has a slightly ambiguous/hazy meaning to me. I'm rationalizing this as:

> If you want to start the commit process and mongosync enabled write blocking for you **but** you disabled it by yourself directly afterwards, you must prevent writes to the source cluster before you start the commit process.

I think what we want is to borrow the wording from the other note in this PR which states that write blocking was never enabled, rather than disabled (because it's not possible for the user to disable something that was never enabled):

> If you want to start the commit process and you did not set `enableUserWriteBlocking` to `true` when you used the `start` endpoint, you must prevent writes to the source cluster before you start the commit process.

Reply:
Done!